request-manager/request/resumer.go

```diff
@@ -37,10 +37,10 @@ type Resumer interface {
 	// been claimed by an RM (`rm_host` field set) but have not been updated in a
 	// while, meaning the RM resuming them probably crashed. These SJCs are
 	// unclaimed (set `rm_host` to null) so they can be resumed in the future. Old
-	// SJCs are those which haven't been resumed within the TTL provided (rounded
-	// to the nearest second). They're deleted and their requests' states set to
-	// FAILED.
-	Cleanup(ttl time.Duration)
+	// SJCs are those which haven't been resumed within the TTL provided when
+	// creating the Resumer (rounded to the nearest second). They're deleted and
+	// their requests' states set to FAILED.
+	Cleanup()
```

Comment:
The TTL was removed but the code comment still references it?

Reply:
The TTL is now provided when creating the Resumer (it's in the ResumerConfig passed to NewResumer), hence "the TTL provided when creating the Resumer".